pkg/disttask/framework/handle/handle_test.go

```diff
@@ -48,7 +48,7 @@ func TestHandle(t *testing.T) {
 	storage.SetTaskManager(mgr)
 
 	// no scheduler registered
-	task, err := handle.SubmitTask(ctx, "1", proto.TaskTypeExample, 2, proto.EmptyMeta)
+	task, err := handle.SubmitTask(ctx, "1", proto.TaskTypeExample, 2, "", proto.EmptyMeta)
```

Comment:
Do we have some test cases for others tidb-service-scope values for example "test1" etc.

Reply:
In TestTargetScope, we generate random test cases when trigger ci.